commitizen/changelog.py

```diff
@@ -281,3 +281,71 @@ def incremental_build(new_content: str, lines: List, metadata: Dict) -> List:
     if not isinstance(latest_version_position, int):
         output_lines.append(new_content)
     return output_lines
+
+
+def get_smart_tag_range(
+    tags: List[GitTag], start: str, end: Optional[str] = None
+) -> List[GitTag]:
+    """Smart because it finds the N+1 tag.
+
+    This is because we need to find until the next tag
+    """
+    accumulator = []
+    keep = False
+    if not end:
+        end = start
+    for index, tag in enumerate(tags):
+        if tag.name == start:
+            keep = True
+        if keep:
+            accumulator.append(tag)
+        if tag.name == end:
+            keep = False
+            try:
+                accumulator.append(tags[index + 1])
+            except IndexError:
+                pass
+            break
+    return accumulator
+
+
+def get_start_and_end_rev(
+    tags: List[GitTag], version: str, tag_format: str, create_tag: Callable
```

Comment:
May I know why do we need to make `create_tag` a `Callable` instead of using the one in `bump`? Are we going to extend it?

Reply:
I also check the function `bump.create_tag`. It looks like `normalize_tag` to me as it does not create a new tag.

Reply:
I renamed it to `normalize_tag`. I guess we can use the one directly in tag 👍🏻 
I don't recall why I was using a callable to be honest 🤔